float64[] joint_position
---
trajectory_msgs/JointTrajectory trajectory_commanded
sensor_msgs/JointState[] trajectory_read
---
trajectory_msgs/JointTrajectoryPoint point_commanded
sensor_msgs/JointState joint_state
float64 time_from_start
